rand_distr/src/normal_inverse_gaussian.rs

```diff
@@ -0,0 +1,82 @@
+use crate::{Distribution, Float, InverseGaussian, Standard, StandardNormal};
+use rand::prelude::*;
```

Comment:
Again, explicit imports would be nicer here.

Reply:
ok